core/client/fs/src/main/java/alluxio/client/block/stream/BlockWorkerDataWriter.java

```diff
@@ -71,7 +71,7 @@ public static BlockWorkerDataWriter create(final FileSystemContext context, long
       BlockWriter blockWriter = blockWorker.createBlockWriter(sessionId, blockId);
       return new BlockWorkerDataWriter(sessionId, blockId, options, blockWriter, blockWorker,
           chunkSize, reservedBytes, conf);
-    } catch (WorkerOutOfSpaceException | IllegalStateException e) {
+    } catch (ResourceExhaustedRuntimeException | IllegalStateException e) {
```

Comment:
I notice you have many irrelevant changes included from other commits. This is typically from how you do merge/rebase. You have 2 options here:
1. Rebase
```
$ git fetch --all
$ git rebase upstream/master
# Resolve all conflicts
$ git push -f origin modify-defaultFS-mount # This will overwrite your commit history with a forced push
```
2. Merge
```
$ git fetch --all
$ git merge upstream/master
# Resolve all conflicts, and this adds one merge commit to your history
$ git push origin modify-defaultFS-mount # Note there's no -f
```
Both work okay for your PR. I'm not sure how you kept up with the `master` that brought those commits into your branch. Normally they are not there (if you do it either way above). Pls remove those irrelevant code changes.

Reply:
Got it.